compose-files/docker-compose.yml

```diff
@@ -207,8 +206,7 @@ services:
       - FILES_STORAGE__S3_BUCKET=schulcloud
       - FILES_STORAGE__S3_REGION=eu-central-1
       # URIs and Port
-      - DB_URL=mongodb://mongodb:27017,mongodb-secondary:27017,mongodb-arbiter:27017/schulcloud?replicaSet=rs0&tls=false&readPreference=secondary
-      - DATABASE__URL=mongodb://server-mongodb:27017/schulcloud
+      - DB_URL=mongodb://mongodb-primary:27017,mongodb-secondary:27017,mongodb-arbiter:27017/schulcloud?replicaSet=rs0&tls=false&readPreference=secondary
```

Comment:
you may also remove `&readPreference=secondary` as this is not what we currently use

Reply:
done